blocks/EnsembleBlocks.js

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
What is this file?

Reply:
looks like I added an extra line at the top by mistake